swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+from swap_meet.vendor import Vendor
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category='', condition = 0, age = 0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return 'Hello World!'
+    
+    def condition_description(self):
+        value = round(self.condition)
+        if value == 5:
+            return "Straight outta da display"
+        if value == 4:
+            return "Basically new"
+        if value == 3:
+            return "Yah it's used, but we be sustainable"
+        if value == 2:
+            return "It ain't THAT bad... maybe just a litte"
+        if value == 1:
+            return "It's intact"
+        if value == 0:
+            return "Just TAKE it!"        
```

Comment:
Nice job defining this once here and having it be inherited by the child classes! I like how you do rounding too!